src/api.rs

```diff
@@ -338,6 +347,9 @@ impl Default for PixelRange {
     }
 }
 
+/// Contains different types of Matrix Coefficients which is used in deriving the luma and chroma (YCbCr) from the R'G'B' primaries or X, Y, Z primaries.
```

Comment:
- R'G'B' is not a thing (it exists but kiwf)
- the matrix coeffs do not covert to ycbcr exclusively, they can covert across a multitude of systems (like ycgco)